recipe/meta.yaml

```
@@ -27,24 +28,31 @@ requirements:
     - cmake
     - ninja
     - make  # [not win]
-    - python  # [win]
+      #- python  # [win]
     - pkg-config  # [not win]
+    - llvm-openmp  # [osx]
   host:
-    - glpk
-    - libxml2
-    - suitesparse
-    - arpack   # [not win]
-    - gmp  # [not win]
-    - mpir  # [win]
-    - libblas
-    - liblapack
+    - glpk 4.65
+    - libxml2 2.10
```

```suggestion
    - libxml2 {{ libxml2 }}
```
As we have it in the generic cbc.yaml

---

yes, thanks